webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-association.js

```diff
@@ -66,13 +68,13 @@ class SakaiRubricAssociation extends RubricsElement {
         ${this.readOnly ? "" : html`
           <div class="radio">
             <label>
-              <input @click="${this.associate}" name="rbcs-associate" type="radio" .value="${this.dontAssociateValue}" ?checked=${!this.isAssociated} ?disabled=${this.readOnly}>${this.dontAssociateLabel}
+              <input @click="${this.associate}" id="dont-associate-value" name="rbcs-associate" type="radio" .value="${this.dontAssociateValue}" ?checked=${!this.isAssociated} ?disabled=${this.readOnly}>${this.dontAssociateLabel}
```

Comment:
This should probably be: do-associate-radio

Reply:
Done.